components/controls/HelpTooltip.tsx

```diff
@@ -20,37 +20,43 @@
 import * as classNames from 'classnames';
 import * as React from 'react';
 import HelpIcon from '../icons/HelpIcon';
+import { IconProps } from '../icons/Icon';
 import { ThemeConsumer } from '../theme';
 import './HelpTooltip.css';
 import Tooltip, { Placement } from './Tooltip';
 
-interface Props {
+interface Props extends Pick<IconProps, 'size'> {
   className?: string;
   children?: React.ReactNode;
   onShow?: () => void;
   overlay: React.ReactNode;
   placement?: Placement;
-  tagName?: string;
 }
 
-export default function HelpTooltip(props: Props) {
-  const { tagName = 'div' } = props;
+export default function HelpTooltip({ size = 12, ...props }: Props) {
+  return (
+    <div className={classNames('help-tooltip', props.className)}>
+      <Tooltip
+        mouseLeaveDelay={0.25}
+        onShow={props.onShow}
+        overlay={props.overlay}
+        placement={props.placement}>
+        <span className="display-inline-flex-center">
+          {props.children || (
+            <ThemeConsumer>
+              {(theme) => <HelpIcon fill={theme.colors.gray71} size={size} />}
+            </ThemeConsumer>
+          )}
+        </span>
+      </Tooltip>
+    </div>
+  );
+}
 
-  return React.createElement(
-    tagName,
-    { className: classNames('help-tooltip', props.className) },
-    <Tooltip
-      mouseLeaveDelay={0.25}
-      onShow={props.onShow}
-      overlay={props.overlay}
-      placement={props.placement}>
-      <span className="display-inline-flex-center">
-        {props.children || (
-          <ThemeConsumer>
-            {(theme) => <HelpIcon fill={theme.colors.gray71} size={12} />}
-          </ThemeConsumer>
-        )}
-      </span>
-    </Tooltip>
+export function DarkHelpTooltip({ size = 12, ...props }: Omit<Props, 'children'>) {
+  return (
+    <HelpTooltip {...props}>
+      <HelpIcon fill="rgba(0,0,0,0.25)" fillInner="white" size={size} />
+    </HelpTooltip>
```

Comment:
NP: Why not add a `dark` flag in the props of HelpTooltip?
NP: Add `rgba(0,0,0,0.25)` & `white` to the theme?

Reply:
1. I don't know actually, I didn't think about it xD I wonder if I should do it with a prop instead ? At least like this we can ignore the children prop which would not work with the dark prop... wdyt ?

2. Ok, I wanted but avoided doing this because I thought it would force you on SQ to add variables you don't care about...

Reply:
1. Ah, I see what you mean. Maybe the problem is the HelpTooltip itself, because if you provide children, it kinda becomes a generic Tooltip :shrug:  
2. We wouldn't need to define them: they would be undefined, but we don't use the new variant anyway, so I think it's fine.

Reply:
1. Ah yeah you are right, there is not much added by the component if you pass a children... But in SC this behavior is used in quite some places... So I'll just leave it like this I think.
2. All good then :) I'll know for next time!